### ISSUE TITLE
[Question] ServiceInfo.addresses does not contain IPv6 addresses

### ISSUE THREAD

**Issue by gaiuscosades** [jstasiak/python-zeroconf#1054]
I am very new to using this excellent library so please close this ticket if this is just nonsense.

I tried to discover Services in a mixed setting IPv4/IPv6 and was astonished that the addresses property of the ServiceInfo that my code discovers and after constuction the ones that it advertises, contain only the IPv4 string. The IPv6 addresses are resolved correctly though and can be obtained via parsed_addresses() or addresses_by_version() just like one would expect.

TL;DR: ServiceInfo.addresses does not contain IPv6 Strings, only the IPv4 Strings.

Is this a bug or well defined library behavior?

**Comment by bdraco** [comment]
This is expected for backwards compatibility
